selenium_tests/CanonizerCampPage.py

```diff
@@ -20,17 +22,29 @@ def load_topic_page(self):
     def load_create_camp_page(self):
         self.load_topic_page()
         # Click on Create New Camp
-        time.sleep(3)
         self.hover(*CreateNewCampPageIdentifiers.CREATE_CAMP)
         self.find_element(*CreateNewCampPageIdentifiers.CREATE_CAMP).click()
-        return CanonizerCampPage(self.driver)
+        title = self.find_element(*CreateNewCampPageIdentifiers.TITLE).text
+        if title == 'Create New Camp':
```

Comment:
I think create variable `new_camp = 'Create New Camp'` and use the variable any where in the code is good